### DISCUSSION TITLE
Location of the gptcode file

### DISCUSSION THREAD

**Discussion by ferencdv** [ricklamers/gpt-code-ui#19] [Q&A]
Hi Rick,
I tried to clone your repo and installed it on Windows 11. However, when I run gptcode it does not seem to find the file. I am in the gpt-code-ui director.  Should I run gptcode in another directory? 
Thanks,
Ferenc

**Comment by ferencdv** [comment]
Never mind it worked fine if I just run python main.py. So for some reason the gptcode does not work. Also, I seem to have an issue with the GPT4 implementation. I put in my key but get an error. It seems to work fine though for GPT3.5. Thanks for the great work!

**Reply by ricklamers** [reply]
GPT-4 API access might not be enabled for your account, see https://openai.com/waitlist/gpt-4-api

**Comment by ricklamers** [comment]
You could just run `pip install gpt-code-ui` to install the package from PyPI without cloning the repo. If you want to install the repo as a local package you can run `pip install -e .` but be sure to build the frontend (`’make compile_frontend`, this requires an up to date `npm`)

**Reply by ekw8724** [reply]
Okay thanks
